### ISSUE TITLE
Variables show in "Related Functions" section

### ISSUE THREAD

**Issue by albert-github** [doxygen/doxygen#9713]
When running the example of "Documentation for elements outside class but inside namespace missed (Origin: bugzilla 155859)"
#1308 we see in the current doxygen master version (1.9.6 (110a7e7bdc5e233e177b2ab1bfd09bae089778b8)):

![image](https://user-images.githubusercontent.com/5223533/205664347-f73b9f82-c053-478f-8f79-336d202ce727.png)

Though in the code we see:
```
  /*!
   *  @relates Examples::ConstInt
   * A constant integer whose documentatioan is taken.
   */
  const int Examples::constIntOutsideClass = 2;
```
so clearly a variable and the documentation for `@relates` states:
> 24.49 \relates `<name>`
>
> This command can be used in the documentation of a non-member function <name>. It puts the function inside the
> 'related function' section of the class documentation. This command is useful for documenting non-friend functions
> that are nevertheless strongly coupled to a certain class. It prevents the need of having to document a file, but only
> works for functions.

So it looks like that that either:
- the documentation is incorrect and also variables should be included, but this would also mean that we would need a new category with "Related Variables"
- the code is wrong and the `\relates` command should not accept variables.

@doxygen what should it be?

Example: [example.tar.gz](https://github.com/doxygen/doxygen/files/10155124/example.tar.gz)

**Comment by doxygen** [comment]
I would opt for supporting variables (and typedefs, and enums) as well. The easiest solution is to change the heading to something more generic, like 'Related Symbols' with note 'These are not members'. An alternative would be a separate section per type.

**Comment by albert-github** [comment]
Easiest would indeed be to change the heading to something like "Related Symbols", makes it easier to add other parts as well. When having a separate section per "type" is would require extra space on the page and a translator function for each "type".
I think that the solution per type in the output would be more consistent with the other doxygen output.

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #9769

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.7.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
